### ISSUE TITLE
ERROR: no path found for ATK_jll [7b86fcea-f67b-53e1-809c-8f1719c154e8] and hash d2cb610e0f31d9d20f4ca37e4535ea5e6f374889

### ISSUE THREAD

**Issue by kafisatz** [timholy/Revise.jl#645]
Recently I randomly run into the error below.
Any idea why?
How could I further debug this? 

I think this may cause this issue https://github.com/julia-vscode/julia-vscode/issues/2358

```
julia> df00 = deepcopy(df)
ERROR: no path found for ATK_jll [7b86fcea-f67b-53e1-809c-8f1719c154e8] and hash d2cb610e0f31d9d20f4ca37e4535ea5e6f374889
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] (::Revise.var"#40#41"{Dict{Base.PkgId,String},String})(::IOStream) at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\pkgs.jl:465
 [3] open(::Revise.var"#40#41"{Dict{Base.PkgId,String},String}, ::String; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at .\io.jl:325
 [4] open at .\io.jl:323 [inlined]
 [5] manifest_paths! at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\pkgs.jl:450 [inlined]
 [6] manifest_paths at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\pkgs.jl:493 [inlined]
 [7] (::Revise.var"#42#43"{String})() at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\pkgs.jl:508
 [8] with_logstate(::Function, ::Any) at .\logging.jl:408
 [9] with_logger at .\logging.jl:514 [inlined]
 [10] watch_manifest(::String) at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\pkgs.jl:505
 [11] (::Revise.TaskThunk)() at C:\Users\bernhard.konig\.julia\packages\Revise\OgnOk\src\types.jl:272



julia> 
  OS: Windows (x86_64-w64-mingw32)
  CPU: AMD Ryzen 9 5950X 16-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, generic)
Environment:
  JULIA_EDITOR = "C:\Program Files\Microsoft VS Code\Code.exe"
  JULIA_NUM_THREADS = 4
julia> 

```

**Comment by timholy** [comment]
I'd need something I can reproduce locally. Does it happen every time? What is `df`?

**Comment by kafisatz** [comment]
A DataFrame. Don't think it is related to that though. 
Is there any additional logging I can enable to help us here?
I currently cannot trigger this intentionally...

**Comment by timholy** [comment]
https://timholy.github.io/Revise.jl/stable/debugging/

**Comment by kafisatz** [comment]
I just sent you an email with logs on this. Although the logs don't seem to contain anything useful as far as I can tell.

**Comment by timholy** [comment]
AFAICT it was the manifest update that triggered this. Presumably the most serious issue is the delayed REPL? I think that may be fixed in #649, but I can't test it.

**Comment by kafisatz** [comment]
Indeed a Manifest update does trigger the issue. 
Interestingly the error message is not always the one from post 1 above.
I have not tested your fix yet (I will wait until it is on master or a release).
